src/aspire/denoising/denoised_src.py

```diff
@@ -47,6 +47,6 @@ def _images(self, start=0, num=np.inf, indices=None, batch_size=512):
         for istart in range(start, end + 1, batch_size):
             imgs_denoised = self.denoiser.images(istart, batch_size)
             iend = min(istart + batch_size, end + 1)
-            im[istart:iend] = imgs_denoised.data
+            im[istart - start : iend - start] = imgs_denoised.data
```

Comment:
If `imgs_denoised` is an `Image` we may want to use the `asnumpy` instead of the data attribute.  

One day I think we will move away from `.data` because it can conflict with numpy's `.data` yielding confusing errors.